liwords-ui/src/about/about.tsx

```diff
@@ -181,14 +181,14 @@ export const About = (props: Props) => {
               <div className="team-info">
                 <h3>Doug Brockmeier</h3>
                 <p>
-                  Doug has been playing Scrabble for over 30 years -- a Guinness
-                  World Record. This makes him feel old, but still dignified.
-                  Early on, his Grandma Elaine taught him the importance of
-                  maximizing Power Tiles through crushing two-way
+                  Doug has been playing Scrabble© for over 30 years -- a
+                  Guinness World Record. This makes him feel old, but still
+                  dignified. Early on, his Grandma Elaine taught him the
+                  importance of maximizing Power Tiles through crushing two-way
                   Double-Letter-Score plays for a whopping 20 points at a time!
                   As a teen, he discovered the club and tournament scene at a
                   local bookstore, where he was welcomed to compete responsibly
-                  for $1 scratchoff jackpots against a wide range of Scrabble
+                  for $1 scratchoff jackpots against a wide range of Scrabble©
```

Comment:
small thing but I think this is actually a registered trademark and not a copyright (so TM or (R)